front/app/modules/commercial/content_builder/admin/components/CraftComponents/WhiteSpace/index.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+
+// components
+import {
+  Box,
+  Radio,
+  Title,
+  Toggle,
+  useBreakpoint,
+} from '@citizenlab/cl2-component-library';
+import Divider from 'components/admin/Divider';
+
+// craft
+import { useNode, UserComponent } from '@craftjs/core';
+
+// intl
+import { FormattedMessage } from 'utils/cl-intl';
+import messages from '../../../messages';
+
+const WhiteSpace: UserComponent = ({ size, withDivider }) => {
+  const isPhone = useBreakpoint('phone');
+
+  const calculatePaddingY = () => {
+    switch (size) {
+      case 'small':
+        return isPhone ? '8px' : '12px';
+      case 'medium':
+        return isPhone ? '16px' : '24px';
+      case 'large':
+        return isPhone ? '24px' : '36px';
+      default:
+        return isPhone ? '8px' : '12px';
+    }
+  };
+  return (
+    <Box id="e2e-white-space" w="100%" paddingY={calculatePaddingY()}>
+      {withDivider && <Divider m="0" />}
+    </Box>
+  );
+};
+
+const WhiteSpaceSettings = () => {
+  const {
+    actions: { setProp },
+    size,
+    withDivider,
+  } = useNode((node) => ({
+    size: node.data.props.size,
+    withDivider: node.data.props.withDivider,
+  }));
+
+  return (
+    <Box mb="30px">
+      <Title variant="h4" as="h3">
+        <FormattedMessage {...messages.whiteSpaceRadioLabel} />
+      </Title>
+      <Radio
+        onChange={(value) => {
```

Comment:
This is a very small personal preference and in no way blocks merging but generally I find it easier if you have multiple similar components in a row like this to have all of the props in the same order, it makes comparing them during a review a little easier

Reply:
Good point 👍